app/styleguide/routes/layout.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div :class="b()">
```

Comment:
- [ ] Please create a `l-default` component from this template and use this here instead. I would also recommend to convert `c-content` to a part of that new layout component.

Reply:
I created a frame component which basically just contains 
<router-view/>
<s-navigation/> 
and just added "l-default" to the routes instead of "layout". Hope that's what you meant.